dmcontent/formats.py

```diff
@@ -2,16 +2,40 @@
 from typing import Optional, Union
 
 
+def comma_format(number: Union[str, float, int]) -> str:
+    """
+    Format a number with commas if it's greater than 999.
+    eg: `1000 -> '1,000'
+        123456.78 -> '123,456.78'
+        '1000000' -> '1,000,000'
+        500 -> '500'
+
+    :param number: The number to be formatted. Either an int, a float or a str.
+    :return: A string of the comma formatted number
+    """
+    if type(number) == str:
```

Comment:
Is it necessary to provide exactly `str`? Could we use [`isinstance`] to allow for subclasses of `str`?

(and ditto for `int` below)

Reply:
That does seem sensible. I'll have a look at where this gets called to check we're not doing anything unusual with types there but if not I'll update